packages/react/src/radio/Radio.css.ts

```diff
@@ -0,0 +1,70 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+
+export const optionWrapper = style({
+  alignItems: "center",
+  flexDirection: "row",
+  gap: "8px",
+  selectors: {
+    ["&:not(:first-child)"]: {
+      marginTop: "12px",
+    },
+  },
+});
+
+export const RadioGroupItem = style({
```

Comment:
styles should not be `PascalCased` and should be `camelCased` instead - and always preferably a single word
```suggestion
export const item = style({
```